src/main/java/cms/gov/madie/measure/resources/ExportController.java

```diff
@@ -82,4 +82,32 @@ public ResponseEntity<byte[]> getTestCaseExport(
     return fhirServicesClient.getTestCaseExports(
         measure, accessToken, testCaseId, bundleType.orElse(("COLLECTION").toUpperCase()));
   }
+
+  @GetMapping(path = "/measures/{id}/qdm/qrda", produces = "application/zip")
+  public ResponseEntity<byte[]> getQRDA(
+      Principal principal,
+      @PathVariable("id") String id,
```

Comment:
okay, QRDA service is going to need cqm measure(converted MADiE measure to cqmmeasure) and calculation results. I'm assuming there is a separate story to get the actual export

Reply:
@adongare Yeah that will be done in the Ruby web service.

Reply:
got it.